menu/navigation.json

```diff
@@ -336,24 +336,20 @@
               {
                 "items": [
                   {
-                    "label": "Invite a user to an Organization",
+                    "label": "Invite a Guest to an Organization",
                     "slug": "invite-user-to-orga"
```

Comment:
can't the slug also be renamed `invite-guest-to-orga`?

Reply:
no because it's an existing page, we would have to make redirections/check for backlinks. as long as the name of the page and the menu entry is correct, it's ok. 🙏🏻